src/components/molecules/help-unit-elements/UnitComment.tsx

```diff
@@ -188,6 +194,31 @@ const UnitComment = ({ id }: BoardIdType) => {
     isLogin && getMyProfileApi();
   }, []);
 
+  const handleChatIconClick = async ({
+    mentorId,
+    mentorName,
+  }: handleChatIconClickType) => {
+    if (isLogin === false) {
```

Comment:
```suggestion
    if (isLogin) {
```
애초에 불리안값이라 그냥해도 괜찮을 듯

Reply:
수정완료